README.md

```diff
@@ -11,11 +11,17 @@ A more detailed description is available in the Requirements Document under the
 
 Follow the instructions in the [Laravel documentation](https://laravel.com/docs/5.6/installation) to install required dependencies and the Laravel package itself.
 
-Clone the repository and create your own `.env` file by copying `.env.example` and providing necessary data. Then run these commands inside the repository:
+Clone the repository and create your own `.env` file by copying `.env.example` and providing necessary data. Please provide Evidencio API key here as well. Then run these commands inside the repository:
 
     composer install
```

Comment:
You probably want to create a bash script that does all this for you such that the user can just run `./install`

Reply:
Should the script also ask for the data to be inputted into .env and automatically create the file?

Reply:
Not necessarily, whatever you prefer. In its simplest form it would just be `composer install && php artisan key:generate && php artisan migrate.`